### ISSUE TITLE
Optimize the size of GAE deployment

### ISSUE THREAD

**Issue by Mistobaan** [studioml/studio#221]
6271 files and 245MB seems a lot for such a project :) 

![_gitignore_ _studio](https://user-images.githubusercontent.com/112599/30780639-becb22a0-a0c4-11e7-9642-e40938e3bcf6.png)

**Comment by pzhokhov** [comment]
Fair point... But I think most of the files and size are coming from various libraries (not from the studio code itself). So what we can do is have a separate requirements_server.txt file that has only the requirements necessary for the server to function, and use that to deploy the app.
